src/NexusMods.App.UI/Controls/LoadoutCard/Standard/LoadoutCardViewModel.cs

```diff
@@ -0,0 +1,130 @@
+                .OnUI()
+                .BindToVM(this, x => x.LoadoutName)
+                .DisposeWith(d);
+            
+            applyService.LastAppliedRevisionFor(loadout.InstallationInstance)
+                .Select(rev => rev.Id == loadout.LoadoutId)
+                .OnUI()
+                .BindToVM(this, x => x.IsLoadoutApplied)
+                .DisposeWith(d);
+
+            var interval = Observable.Interval(TimeSpan.FromMinutes(1)).Prepend(0);
+            
+            interval.Select(_ => FormatCreatedTime(loadout.CreatedAt))
+                .OnUI()
+                .BindToVM(this, x => x.HumanizedLoadoutCreationTime);
+            
+            // TODO: implement getting LastApplied time by updating Apply detection code to use Revision changes instead of tx changes 
+            // interval.Select(_ => FormatCreatedTime(loadout.LastAppliedAt))
+            //     .OnUI()
+            //     .BindToVM(this, x => x.HumanizedLoadoutLastApplyTime);
+
+            Loadout.Observe(conn, loadout.Id)
+                .Select(l => FormatNumMods(l.Mods.Count))
+                .OnUI()
+                .BindToVM(this, x => x.LoadoutModCount)
+                .DisposeWith(d);
+
+        });
+        
+    }
+
+    public ILoadoutBadgeViewModel LoadoutBadgeViewModel { get; private set; }
+    [Reactive] public string LoadoutName { get; private set; }
+    [Reactive] public IImage? LoadoutImage { get; private set; } 
+    [Reactive] public bool IsLoadoutApplied { get; private set; } = false;
+    [Reactive] public string HumanizedLoadoutLastApplyTime { get; private set; } = "";
+    [Reactive] public string HumanizedLoadoutCreationTime { get; private set; } = "";
+    [Reactive] public string LoadoutModCount { get; private set; } = "";
+    [Reactive] public bool IsDeleting { get;  private set; } = false;
+    public bool IsSkeleton => false;
+    public required ReactiveCommand<Unit, Unit> VisitLoadoutCommand { get; init; }
+    public required ReactiveCommand<Unit, Unit> CloneLoadoutCommand { get; init; } 
+    public ReactiveCommand<Unit, Unit> DeleteLoadoutCommand { get; }
+    
+    
+    private string FormatNumMods(int numMods)
+    {
+        return string.Format(Language.LoadoutCardViewModel_FormatNumMods_Mods__0_, numMods);
+    }
+    
+    private string FormatCreatedTime(DateTime creationTime)
+    {
+        return string.Format(Language.LoadoutCardViewModel_CreationTimeConverter_Created__0_, creationTime.Humanize());
+    }
+    
+    private string FormatLastAppliedTime(DateTime creationTime)
+    {
+        return string.Format(Language.LoadoutCardViewModel_FormatLastAppliedTime_Last_applied__0_, creationTime.Humanize());
+    }
+    private async Task<Bitmap?> LoadImage(GameInstallation source)
+    {
+        return await Task.Run(async () =>
+        {
+            try
+            {
+                var stream = await source.GetGame().Icon.GetStreamAsync();
+                return new Bitmap(stream);
```

Comment:
We'd probably wanna cache this at some point.

Reply:
Agreed, as we discussed this should probably happen on the Game side since we use this in a lot of places. 
Created issue for it: #1797